Example/Messaging/App/watchOS/SampleWatchApp/SampleWatchApp WatchKit Extension/ExtensionDelegate.swift

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import WatchKit
+
+import FirebaseCore
+import FirebaseMessaging
+
+class ExtensionDelegate: NSObject, WKExtensionDelegate, MessagingDelegate {
```

Comment:
nit: class docs?

Reply:
You mean link the API of MessagingDelegate here?

Reply:
No, more like, docs that tell me what the ExtensionDelegate class is and will be used for

Reply:
do you mean adding the link to the MessagingDelegate API?

Reply:
This is default class generated by watch extension

Reply:
Oh ok--I haven't written watch apps so that's not apparent to me.

Reply:
yeahit's like an app delegate thing. I will add a comment on top to clarify that.